src/components/Menu/MenuItem/menuItem.module.scss

```diff
@@ -116,10 +121,16 @@
     }
   }
 
-  &[disabled] {
-    pointer-events: none;
+  &.disabled {
+    cursor: not-allowed;
     opacity: $disabled-alpha-value;
+    pointer-events: none;
```

Comment:
if we have `pointer-events: none`, do we still need the preventDefault?

Reply:
Yes, easily changed in the web inspector.